src/schemas/room.ts

```diff
@@ -1,3 +1,9 @@
+export enum RoomType {
+  UNKNOWN = 0,
```

Comment:
Please only keep the first letter to be capitalized, because, in our code base, all enum names follow this format.

UNKNOWN -> Unknown